concourse/pipelines/deploy.yml

```diff
@@ -194,6 +194,11 @@ jobs:
       params:
         APPLICATION: publisher
         GOVUK_ENVIRONMENT: test
+    - task: update-worker-task-definition
+      file: govuk-infrastructure/concourse/tasks/update-task-definition.yml
+      params:
+        APPLICATION: publisher-worker
```

Comment:
Another drive by comment by me. Should we think about naming other services with a `-web` suffix to indicate their usage? It might be beneficial approach to avoid making it looking like one application is the de-facto and the other is minor. Similar to how you'd approach a Procfile: 

https://github.com/alphagov/content-publisher/blob/2f91d43bcd1eb1f2d30f9d1ff9e556cd4d0de2a6/Procfile#L1-L2

Reply:
@kevindew thanks for your suggestion. I've changed the publisher ECS Service and container name to "publisher-web" and I've made similar changes to the directories. See the more recent commit.

Reply:
Nice one, thanks @bilbof